src/onnx-mlir.cpp

```diff
@@ -54,6 +54,9 @@ int main(int argc, char *argv[]) {
   mlir::registerPassManagerCLOptions();
   mlir::registerDefaultTimingManagerCLOptions();
 
+  llvm::cl::AddExtraVersionPrinter(
```

Comment:
All of the llvm tools specify their versions first before adding llvm. I looked at the code in more detail and the duplication is very little, so I think the right approach is to follow that pattern and have onnx-mlir version than llvm version similar to how clang, lldb, etc. do it.

Reply:
ok, I can duplicate some code, and have onnx-mlir show up first.